etc/script/build.sh

```
@@ -1,57 +1,79 @@
-#color code
+# Color codes
 C_RED="\033[31m"
 C_GREEN="\033[32m"
-
 C_END="\033[0m"
 
-BUILD_TYPE=Release
+# Build configuration
+K_VERSION="1.0.0"
+BUILD_TYPE="Release"
 VERBOSE=0
```

Shellcheck (0.10.0)

[warning] VERBOSE appears unused. Verify use (or export if used externally).

(SC2034)